content/docs/2.5/scalers/newrelic.md

```diff
@@ -0,0 +1,70 @@
++++
+title = "Newrelic"
+availability = "2.5+"
+maintainer = "Community"
+description = "Scale applications based on NewRelic NRQL"
+layout = "scaler"
+go_file = "newrelic_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `Newrelic` trigger that scales based on a Newrelic metric.
+
+```yaml
+triggers:
+  - type: newrelic
+    metadata:
+      # Required: nrAccount - Subaccount to run the query on
+      nrAccount: 1234567
+      # Required: nrQueryKey - Api key to connect to New Relic
+      nrQueryKey: "NRAK-xxxxxxxxxxxxxxxxxxxxxxxxxxx"
+      # Required: nrRegion - Region to query data for
+      nrRegion: "US"
+      # Required: metricName
+      metricName: "duration"
+      # Required: nrql
+      nrql: "SELECT average(duration) from Transaction where appName='SITE' TIMESERIES"
+      # Required: threshold
+      threshold: 100
+      # Optional: nrLogLevel
+      nrLogLevel: "info"
+```
+
+**Parameter list:**
+
+- `nrAccount` - The account within Newrelic that the request should be targeted against.
+- `nrQueryKey` - The API key that will be leveraged to connect to Newrelic and make requests. [official documentation](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/)
+- `nrRegion` - The region to connect to for the Newrelic apis. (Values: `LOCAL`, `EU`, `STAGING`, `US`, Default: `US`)
```

Comment:
Is this optional? I mean, if it has default value I feel it as optional, WDYT?

Reply:
Yep good callout given that there's a default its optional